astropy/utils/tests/test_data.py

```diff
@@ -98,19 +107,20 @@ def test_find_by_hash():
 
 
 # Package data functions
-@pytest.mark.parametrize(('filename'), ['local.dat', 'local.dat.gz', 'local.dat.bz2'])
+@pytest.mark.parametrize(('filename'), ['local.dat', 'local.dat.gz', 'local.dat.bz2', 'local.dat.xz'])
 def test_local_data_obj(filename):
```

Comment:
I was able to confirm this locally, so I don't think it's just a transient failure.  And it is indeed happening when `'local.dat.xz'` is passed to the filename fixture.